spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/CachingConnectionFactory.java

```diff
@@ -386,7 +386,7 @@ public void setConnectionLimit(int connectionLimit) {
 
 	@Override
 	public boolean isPublisherConfirms() {
-		return ConfirmType.CORRELATED.equals(this.confirmType);
+		return !ConfirmType.NONE.equals(this.confirmType);
```

Comment:
This is a behavior change in the patch version.
So, we are good with the fix, but only when we start a new minor `3.2` version in May.

Might be grate if you prove the fix with some tests meanwhile.

Thanks

Reply:
OK